### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by jcompagner** [Eonasdan/tempus-dominus#2689]
### Prerequisites

- [X] I have [searched](https://github.com/Eonasdan/tempus-dominus/issues?q=label%3A%22Type%3A+feature+request%22+) for duplicate or closed feature requests.
- [X] I have read the [contributing guidelines](https://github.com/Eonasdan/tempus-dominus/blob/master/.github/CONTRIBUTING.md).

### Proposal

we have kind of a patch on top of it implemented:

https://github.com/Servoy/bootstrapcomponents/commit/5fef17ab2f9340f965ed8cf15053d3a016f3141a#diff-7834e0d054a2cb977d1a07b4a35a36fa7f1bd4992001fe8551ffaa39103b7b21R31

but it sounds ot me that this kind of keyboard navigation should be in the core product

### Motivation and context

keyboard navigation is quite important to do stuff
like arrow keys to go over the days or delete key to delete the value, esc to close the popup

which is for fast entry sometimes way faster then using the mouse

There is not something yet in 6 right? i think it was in the older version.

**Comment by Eonasdan** [comment]
I'm going to close this in favor of the #2390. I do plan to add the keyboard nav back but I feel like there's some "at odds" requests. The arrows keys can't be used to navigate the input field AND the picker for instance.
